src/modules/Margin.ts

```diff
@@ -493,7 +493,22 @@ export default class Margin {
     return this.contracts.margin.isLoanApproved.call(loanHash);
   }
 
-  public async getAllPositionClosedEvents(
```

Comment:
this is a breaking change because you're renaming this function. So any other repos that use this function will break. I suggest just keeping the same name (with "All) so it's not breaking

Reply:
good call.